src/main/java/mate/academy/dao/jdbc/ManufacturerDaoJdbc.java

```diff
@@ -0,0 +1,112 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.ManufacturerDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Manufacturer;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoJdbc implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturer (name_manufacturer, country_manufacturer) "
+                + "VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query, Statement
+                        .RETURN_GENERATED_KEYS)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            manufacturer = createManufacturer(resultSet);
```

Comment:
you don't need to parse a whole model and it's better to add check for next in order to avoid NPE 
```suggestion
                        if (resultSet.next()) {
                manufacturer.setId(resultSet.getLong(1));
            }
```